spec/presenters/sufia/collection_presenter_spec.rb

```diff
@@ -58,4 +58,9 @@
     subject { presenter.total_items }
     it { is_expected.to eq 0 }
   end
+
+  describe "#identifier" do
+    subject { presenter.identifier }
```

Comment:
I'm just following the test convention that already there. Although, yes, using a syntax expecting delegation would be more "unit-y" and less "integrate-y" but we'd need to make that change throughout the application. Do you want to ticket that for a refactor?